proposal: add custom commit message example (s) (#18302)
github/docs: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

      prefix-development: "pip dev"
      include: "scope"
```

> Example:

Reviewer: ```suggestion
```

> Using these preferences, bumping the `requests` library in the `pip` Development dependency group generates a commit message of:

Reviewer: ```suggestion
If you use the same configuration as the example above, bumping the `requests` library in the `pip` development dependency group will generate a commit message of:
```

>    pip dev: bump requests from 1.0.0 to 1.0.1

Reviewer: ```suggestion
   'pip dev: bump requests from 1.0.0 to 1.0.1'
```

> 

Reviewer: Remove new line
```suggestion
```

      prefix-development: "pip dev"
      include: "scope"
```

> An example commit message based on your recent commit preferences would be `<prefix> (dev-deps): bump requests from 1.0.8 to 1.1.3`.

Reviewer 2: It looks like your example is based on your own personal settings, and not the settings in the above example config, which creates confusion.

IIUC, you've got `(dev-deps)` as the development prefix? It's unclear to me if your prefix includes the `(`,`)`, or if this is auto-added by Dependabot. 

Also, from the docs above it looks like the `prefix-development` will override the `prefix` entirely, not just be two prefixes in a row.

If the above config results in `pip dev: bump requests from 1.0.0 to 1.0.1` consider rewording to:
```suggestion
Example:
Using these preferences, bumping the `requests` library in the `pip` Development dependency group generates a commit message of:

   pip dev: bump requests from 1.0.0 to 1.0.1
   
```

But if I'm misunderstanding something with how this works, please clarify... or just take what I wrote above and tweak further.

Also, note to the Docs repo owners, my markdown formatting may be incorrect, or if you have a specific format for examples, then please fixup.

Author: Hey @jeffwidman , Yesss this is perfect !. Yea some of the prs have brackets.. for example: https://github.com/LearningTypeScript/projects/pull/31
